php/specimendao.class.inc

```diff
@@ -379,6 +381,7 @@ class SpecimenDAO extends \LORIS\Data\ProvisionerInstance
             foreach ($row as $value) {
                 $pA[$row['protocolId']][$row['attributeId']]['label']           = $row['label'];
                 $pA[$row['protocolId']][$row['attributeId']]['datatypeId']      = $row['datatypeId'];
+                $pA[$row['protocolId']][$row['attributeId']]['refTableId']      = $row['refTableId'];
```

Comment:
@ridz1208 do you recall the motivation for this change? It doesn't seem related to anything else in the pR